vulcanai/models/metrics.py

```diff
@@ -598,6 +600,7 @@ def _run_test_single_continuous(network, data_loader, transform_outputs,
             results : dict
 
         """
+        # TODO: this may not work with current TabularDataset
         targets = np.array([v[1] for v in data_loader.dataset])
```

Comment:
If you're referring to the matrix stored version within Tabular Dataset, wouldn't that be easier to extract?

Reply:
yeah, this'll be resolved as long as we convert it to that. I'll take the comment out.